.ci/tidy_script.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 CLANG_TIDY_CHECKS='-*'
 if [[ $TOOL == clang-tidy-analyzer ]]; then
-  CLANG_TIDY_CHECKS+=',clang-analyzer-*,-clang-analyzer-alpha*,bugprone*'
+  CLANG_TIDY_CHECKS+=',clang-analyzer-*,-clang-analyzer-alpha*,-clang-analyzer-cplusplus.NewDelete*,-clang-analyzer-cplusplus.InnerPointer,bugprone*'
```

Comment:
The commit message is confusing to me. This change enables more checks but the commit log says suppressing warnings.

Reply:
Note the "-" in front. That means those checks are disabled.

Reply:
Why did you have to suppress them?

Reply:
Those particular warnings were being thrown on this line - https://github.com/root-project/root/blob/master/core/dictgen/src/rootcling_impl.cxx#L4849 because of the delete.
For eg here - https://travis-ci.org/reikdas/root/jobs/631619634#L3768

Reply:
As we clarified via IM, this was a real issue in rootcling and we have decided to fix that instead.